.github/ISSUE_TEMPLATE/feature_request.yaml

```diff
@@ -0,0 +1,36 @@
+name: Feature request
+description: Suggest an idea for FreeTube which you would like to see in a future release
+title: "[Feature Request]: "
+labels: "enhancement"
+body:
+  - type: checkboxes
+    attributes:
+      label: Guidelines
+      description: Please ensure you've completed all of the following.
+      options:
+          - label: I have searched the [issue tracker](https://github.com/FreeTubeApp/FreeTube/issues) for a feature request that matches the one i want to file, without success.
+            required: true
+  - type: textarea
+    attributes:
+      label: Problem Description
+      description: Please add a clear and concise description of the problem you are seeking to solve with this feature request.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Proposed Solution
+      description: Describe the solution you'd like in a clear and concise manner.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Alternatives Considered
+      description: A clear and concise description of any alternative solutions or features you've considered. 
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Additional Information
```

Comment:
I would change this to: Screenshots / Additional Information

Reply:
To keep it uniform with the bug report template i think that this isnt a good idea.

https://github.com/FreeTubeApp/FreeTube/pull/1593#discussion_r715022409